packages/amplication-data-service-generator/src/admin/create-dotenv.template.env

```diff
@@ -1,2 +1,10 @@
 PORT=3001
-SKIP_PREFLIGHT_CHECK=true
\ No newline at end of file
+SKIP_PREFLIGHT_CHECK=true
+POSTGRESQL_USER=${dbUser}
+POSTGRESQL_PASSWORD=${dbPassword}
+POSTGRESQL_PORT=${dbPort}
+POSTGRESQL_URL=postgres://${dbUser}:${dbPassword}@${dbHost}:${dbPort}${dbName}
+BCRYPT_SALT=10
+COMPOSE_PROJECT_NAME=amp_${appId}
+JWT_SECRET_KEY=Change_ME!!!
+JWT_EXPIRATION=2d
```

Comment:
```suggestion
COMPOSE_PROJECT_NAME=amp_admin_${appId}
```

Reply:
I'm missing the server URL.

the client and server will run on a different port and the client currently will try to connect to the same port that he is running on.